jackson/block-template-parts/footer.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"20px";"bottom":"20px"}}}} -->
```

Comment:
This footer looks very similar to the one that ships in Blockbase. Why do we need a different one?

Reply:
I had issues with spacing - it used to be that the credit in Blockbase sat uncomfortably close to the bottom of the page, but I'm not seeing it anymore, so it can be removed.